Content.Shared/Clumsy/ClumsySystem.cs

```diff
@@ -0,0 +1,131 @@
+using Content.Shared.Chemistry.Hypospray.Events;
+using Content.Shared.Medical;
+using Content.Shared.Weapons.Ranged.Events;
+using Content.Shared.Climbing.Events;
+using Robust.Shared.Random;
+using Robust.Shared.Audio.Systems;
+using Robust.Shared.Audio;
+using Content.Shared.Popups;
+using Content.Shared.Stunnable;
+using Content.Shared.Damage;
+using Robust.Shared.Timing;
+using Content.Shared.IdentityManagement;
+using Content.Shared.CCVar;
+using Content.Shared.Climbing.Components;
+using Robust.Shared.Configuration;
+
+namespace Content.Shared.Clumsy;
+
+public sealed class ClumsySystem : EntitySystem
+{
+    [Dependency] private readonly IRobustRandom _random = default!;
+    [Dependency] private readonly SharedStunSystem _stun = default!;
+    [Dependency] private readonly SharedAudioSystem _audioSystem = default!;
+    [Dependency] private readonly SharedPopupSystem _popup = default!;
+    [Dependency] private readonly DamageableSystem _damageableSystem = default!;
+    [Dependency] private readonly IGameTiming _timing = default!;
+    [Dependency] private readonly IConfigurationManager _cfg = default!;
+
+    public override void Initialize()
+    {
+        SubscribeLocalEvent<ClumsyComponent, SelfBeforeHyposprayInjects>(BeforeHyposprayEvent);
+        SubscribeLocalEvent<ClumsyComponent, SelfBeforeDefibrillatorZaps>(BeforeDefibrillatorZapsEvent);
+        SubscribeLocalEvent<ClumsyComponent, SelfBeforeGunShotEvent>(BeforeGunShotEvent);
+        SubscribeLocalEvent<ClumsyComponent, SelfBeforeClimbEvent>(OnBeforeClimbEvent);
+    }
+
+    // If you add more clumsy interactions add them in this section!
+    #region Clumsy interaction events
+    private void BeforeHyposprayEvent(Entity<ClumsyComponent> entity, ref SelfBeforeHyposprayInjects args)
+    {
+        // Clumsy people sometimes inject themselves! Apparently syringes are clumsy proof...
+        if (!_random.Prob(entity.Comp.ClumsyDefaultCheck))
+            return;
```

Comment:
This shouldn't be done on the client since the result of `_random.Prob` could be different from the server and mispredict. The only reason it works right now is because the event is never actually raised on the client, but it could be in the future. Just return early on client.

Same for the other event handlers.

Reply:
Yeah so I'm not quite sure what I should do here:
1.) Make it prediction ready (Use the the current time for the seed) and leave it in shared
2.) Move it to server and keep it as is.

I was going to do one of those two things but I guess I forgot 😆! What do you think?